### PR TITLE
Add dump_datamodel.py global all command

### PR DESCRIPTION
**Pull request by gregertw**
Gives you the ability to dump any data models, also global system models 
with dump_datamodel.py <all|global> <all|model_name> target_dir

### CODE REVIEW

File: {{cookiecutter.buildfolder}}/dump_datamodel.py [hunk 1]
```diff
@@ -23,10 +23,28 @@
...
+            model_name = None
+        else:
+            model_name = sys.argv[2]
+        dump_datamodels_all(
+            ToolGlobals,
+            target_dir=sys.argv[3],
+            model_name=model_name,
+            include_global=include_global,
+        )
+        exit(0)
```
Comment (andeplane) [inline {{cookiecutter.buildfolder}}/dump_datamodel.py:47]: No big deal, but there are some quite nice libraries for this

File: {{cookiecutter.buildfolder}}/utils/datamodel.py [hunk 2]
```diff
@@ -291,7 +335,7 @@ def dump_datamodel(
             "dataModelInstancesAcl": ["READ", "WRITE"],
         }
     )
-    print("Loading data model ({model_name}) in space ({space_name})...")
+    print(f"Loading data model ({model_name}) in space ({space_name})...")
```
Comment (andeplane) [inline {{cookiecutter.buildfolder}}/utils/datamodel.py:338]: Nice :D

**Review by andeplane — APPROVED** [review]
LGTM

File: {{cookiecutter.buildfolder}}/dump_datamodel.py (hunk 1 above)
Comment (gregertw) [reply {{cookiecutter.buildfolder}}/dump_datamodel.py:47]: Yeah, I know. I just hate adding dependenices for super-simple things. Could have used click, but I think we need to iterate on this and as we discussed, push this into the config/dump service.

**Pull request merged**